pallets/parachain-staking/src/lib.rs

```diff
@@ -438,9 +445,51 @@ pub mod pallet {
 				weight = weight.saturating_add(T::OnNewRound::on_new_round(round.current));
 				// pay all stakers for T::RewardPaymentDelay rounds ago
 				weight = weight.saturating_add(Self::prepare_staking_payouts(round.current));
+
+				// select total collators
+				let collators = <SelectedCandidates<T>>::get();
```

Comment:
This DB read is not accounted

Reply:
It is accounted at the end of `on_initialize()`